CRM/Event/Tokens.php

```diff
@@ -205,12 +203,16 @@ protected function getEventTokenValues(int $eventID = NULL): array {
           'custom.*',
         ], $this->getExposedFields()))
         ->execute()->first();
-      $tokens['location']['text/plain'] = \CRM_Utils_Address::format([
-        'street_address' => $event['loc_block_id.address_id.street_address'],
-        'city' => $event['loc_block_id.address_id.city'],
+      $addressValues = [
+        'address_name' => $event['loc_block_id.address_id.name'],
         'state_province' => $event['loc_block_id.address_id.state_province_id:label'],
```

Comment:
County doesn't work, assume we need a similar line to get that label.

Reply:
@larssandergreen I just added country & county (although I haven't tested them yet)

Reply:
I think better to leave country out. I can't imagine a context where you actually want to print the country in the event receipt (presumably registrants know which country the event is in) and you can't select a province without selecting a country, so users can't stop it from displaying if they need a province.